libs/@hashintel/petrinaut/src/core/types/sdcpn.ts

```diff
@@ -31,13 +29,13 @@ export type Place = {
   height?: number;
 };
 
-export type SDCPNType = {
+export type Color = {
```

Comment:
I agree that it's better to use `Color` rather than `Type` here, but maybe keeping the prefix would help understanding that this does not represent an actual screen color.

```suggestion
export type SDCPNColor = {
```

Again just a remark.

Reply:
We need to figure out a good name for this – since Petrinaut will handle other types of PNs, we want to avoid calling everything 'SDCPNx'. We agreed to come back to this naming later.